packages/frontend-2/pages/projects/[id]/index.vue

```diff
@@ -13,11 +13,6 @@
         <ProjectPageHeader :project="project" />
         <div class="flex gap-x-3 items-center justify-between">
           <div class="flex flex-row gap-x-3">
-            <CommonBadge rounded :color-classes="'text-foreground-2 bg-primary-muted'">
```

Comment:
Not really related to the initial goal of this PR but I also removed this model counter on the project page since there's already a count outputted in the tabbed nav.